sdk/attestation/azure-security-attestation/inc/azure/attestation/attestation_client_models.hpp

```diff
@@ -0,0 +1,122 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+/**
+ * @file
+ * @brief Defines the Azure Attestatoin API types.
+ *
+ */
+
+#pragma once
+
+#include "azure/attestation/dll_import_export.hpp"
+
+#include <azure/core/context.hpp>
+#include <azure/core/http/http.hpp>
+#include <azure/core/nullable.hpp>
+#include <azure/core/paged_response.hpp>
+#include <azure/core/response.hpp>
+#include <iostream>
+#include <memory>
+#include <string>
+#include <unordered_map>
+#include <vector>
+
+namespace Azure { namespace Security { namespace Attestation {
+  class AttestationClient;
+  /**
+   * @brief Contains identity and other basic properties of a Certificate.
+   *
+   */
+  struct AttestationOpenIdMetadata final
+  {
+    /**
+     */
+    ::std::string Issuer;
```

Comment:
replace `::std` to `std`.  Or make all ocurrences of the file to follow `::std`  .  We don't use it like that, anyway

Reply:
IIRC I added the std because of a compilation error earlier. Not seing it now though.